lib/karafka/routing/builder.rb

```diff
@@ -67,6 +67,19 @@ def draw(&block)
         end
       end
 
+      # Clear out the drawn routes.
+      alias array_clear clear
```

Comment:
why 2 names?

Reply:
Ah I get it now!

Since it's used internally only could we maybe make it private not to "expose" a false API?